internal/grpc/handler_test.go

```diff
@@ -0,0 +1,112 @@
+package grpc
+
+import (
+	"context"
+	"testing"
+
+	"github.com/emmaLP/gs-software-onboarding/internal/caching"
+	commonModel "github.com/emmaLP/gs-software-onboarding/pkg/common/model"
+	pbMock "github.com/emmaLP/gs-software-onboarding/pkg/grpc/proto"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMethods(t *testing.T) {
```

Comment:
I think I'd break this into individual method tests

Reply:
I was being lazy so I wouldn't have pretty much duplicate code when the only real difference if the method call